### ISSUE TITLE
[Bug] Make grouped object scalable and rotateable

### ISSUE THREAD

**Issue by huchenlei** [huchenlei/sd-webui-openpose-editor#14]
Following minimal reproduction shows that fabric js support group to be scaled and rotated. For some unknown reason in the openpose editor, once keypoints and connections are grouped, they become unscalable and unrotateable. 

```
<!DOCTYPE html>
<html>
<head>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fabric.js/5.3.1/fabric.min.js"></script>
</head>
<body>
    <canvas id="canvas" width="800" height="600"></canvas>
    <script>
        var canvas = new fabric.Canvas('canvas');

        // Create the first circle
        var circle1 = new fabric.Circle({
            left: 100,
            top: 100,
            fill: 'red',
            radius: 50
        });

        // Create the second circle
        var circle2 = new fabric.Circle({
            left: 200,
            top: 200,
            fill: 'blue',
            radius: 50
        });

        // Group the two circles together
        var group = new fabric.Group([circle1, circle2]);

        // Add the group to the canvas
        canvas.add(group);
    </script>
</body>
</html>
```

**Comment by huchenlei** [comment]
The issue comes from following code:
```
    /**
     * Determines which corner has been clicked
     * @private
     * @param {Object} pointer The pointer indicating the mouse position
     * @return {String|Boolean} corner code (tl, tr, bl, br, etc.), or false if nothing is found
     */
    _findTargetCorner: function(pointer, forTouch) {
      // objects in group, anykind, are not self modificable,
      // must not return an hovered corner.
      if (!this.hasControls || this.group || (!this.canvas || this.canvas._activeObject !== this)) {
        return false;
      }
      // More logic omitted ...
     }
```

`this.canvas._activeObject` is somehow set to a `Proxy` of the underlying object, while `this` is obviously not a `Proxy`.

**Comment by huchenlei** [comment]
https://github.com/vuejs/core/issues/3236

**Comment by huchenlei** [comment]
Fixed in v0.1.0.
